actors/abi/sector.go

```diff
@@ -28,7 +30,11 @@ type SectorID struct {
 type SectorWeight int64 // TODO big
 
 // The unit of storage power (measured in bytes)
-type StoragePower int64 // TODO big
+type StoragePower = big.Int
+
+func NewStoragePower(n int64) StoragePower {
+	return StoragePower(big.NewInt(n))
+}
```

Comment:
Made this a BigInt now so that I would get the marshaling code for free. An alternative was to implement the marshaling interface on it with panics and then replace it later. I thought the former was more reasonable.